docs/networkfence.md

```diff
@@ -0,0 +1,32 @@
+# NetworkFence
+
+NetworkFence is a cluster-scoped custom resource that allows Kubernetes to invoke "Network fence" operation on a storage provider.
+
+The user needs to specify the kind of operation they want to perform (fence/unfence), and then provide the list of CIDR blocks on which the mentioned operation is to be performed.
```

Comment:
here is `list of CIDR blocks`, but it is not a list in the example.

Reply:
Nice catch! The description was old and was not updated. Updated now :)